pkg/operator/csi/csidrivercontrollerservicecontroller/helpers.go

```diff
@@ -215,6 +215,14 @@ func WithPlaceholdersHook(configInformer configinformers.SharedInformerFactory)
 		logLevel := loglevel.LogLevelToVerbosity(spec.LogLevel)
 		pairs = append(pairs, []string{"${LOG_LEVEL}", strconv.Itoa(logLevel)}...)
 
+		// FIPS
+		fipsEnabled := "\"false\""
+		contents, err := os.ReadFile("/proc/sys/crypto/fips_enabled")
+		if err == nil && string(contents) == "1\n" {
+			fipsEnabled = "\"true\""
+		}
+		pairs = append(pairs, []string{"${FIPS_ENABLED}", fipsEnabled}...)
+
```

Comment:
Since this is specific to EFS (so far), have you considered having this in the operator repo instead?

Here's an EBS example [1] used as a DeploymentHook [2]. If we really want to use a ManifestHook [3], we can plumb them through here [4].


[1] https://github.com/bertinatto/aws-ebs-csi-driver-operator-1/blob/10637f68e0e7a3da1b37781ff8f5bc68022dbf5f/pkg/operator/starter.go#L523-L530
[2] https://github.com/bertinatto/aws-ebs-csi-driver-operator-1/blob/10637f68e0e7a3da1b37781ff8f5bc68022dbf5f/vendor/github.com/openshift/library-go/pkg/operator/deploymentcontroller/deployment_controller.go#L27
[3] https://github.com/bertinatto/aws-ebs-csi-driver-operator-1/blob/10637f68e0e7a3da1b37781ff8f5bc68022dbf5f/vendor/github.com/openshift/library-go/pkg/operator/deploymentcontroller/deployment_controller.go#L31
[4] https://github.com/bertinatto/aws-ebs-csi-driver-operator-1/blob/10637f68e0e7a3da1b37781ff8f5bc68022dbf5f/vendor/github.com/openshift/library-go/pkg/operator/csi/csicontrollerset/csi_controller_set.go#L183